aurora-postgresql-plugin/docs/AuroraPostgres-batchsource.md

```diff
@@ -54,6 +54,9 @@ disabled.
 back from the query. However, it must match the schema that comes back from the query,
 except it can mark fields as nullable and can contain a subset of the fields.
 
+**Fetch Size:** The number of rows to fetch at a time per split. Larger fetch size can result in faster import, " +
+"with the tradeoff of higher memory usage.
```

Comment:
I think you need to remove `" +\n"` in Markdown

Reply:
done